pgmpy/metrics/__init__.py

```diff
@@ -4,6 +4,7 @@
     correlation_score,
     fisher_c,
     implied_cis,
+    latent_admg,
```

Comment:
Remove this method from `__init__.py`, so that it doens't show as an option to the user.